packages/cashier/src/components/cashier-locked/cashier-locked.jsx

```diff
@@ -235,6 +237,12 @@ const CashierLocked = ({
         icon = 'IcCashierWithdrawalLock';
         title = localize('Withdrawals are locked');
         message = localize('You can only make deposits. Please contact us via live chat for more information.');
+    } else if (is_withdrawal_locked && pa_commision_withdrawal_limit) {
+        icon = 'IcCashierWithdrawalLock';
+        title = localize('Cashier is locked for withdrawals');
+        message = localize(
+            "It seems that you've no commissions to withdraw at the moment. You can make withdrawals once you receive your commissions."
```

Comment:
```suggestion
            'It seems that you've no commissions to withdraw at the moment. You can make withdrawals once you receive your commissions.'
```

Reply:
@hamid-deriv Prettier will format it to use `"` to avoid unnecessary space once you save the file.